### DISCUSSION TITLE
Task View/Multiple Desktop Problem With Explorer Patcher set to Windows 10 style task bar

### DISCUSSION THREAD

**Discussion by DrMartyG** [valinet/ExplorerPatcher#1717] [Q&A]
I use task view to utilize multiple virtual desktops. With Explorer Patcher installed, it will work if I have the taskbar style set to Windows 11, but it will not work if the taskbar style is set to Windows 10, which I would prefer to use, as it restores the toolbars to the taskbar, and I prefer the weather widget with the Windows 10 style taskbar. Is there a way to set Explorer Patcher to the Windows 10 style taskbar yet retain the ability to use Task View and multiple virtual desktops?

**Comment by Bassadin** [comment]
Seems to be a bug. This worked for me before the latest Win 11 update.

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Reply by Bassadin** [reply]
Windows: `10.0.22621 Build 22621.2070`
EP: `22621.1992.56.1`

**Reply by pyrates999** [reply]
After 22621, there should be a dot followed by a number.  Please post that.  Otherwise just take a screen shot of what winver shows you.

**Reply by Bassadin** [reply]
Sorry, updated it. I took the first version from the "PC info" table

**Reply by pyrates999** [reply]
That is a preview update: https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

**Reply by Bassadin** [reply]
Thanks! I have that unticked, but might have clicked on one of those preview update on the update screen. Can I go back to the latest stable version somehow if I already uninstalled the "previous update" files due to storage cleaning reasons?

**Reply by pyrates999** [reply]
Yes you should be able to.  You can tell by running winver and checking the build number.  It should end with .1992

**Reply by DrMartyG** [reply]
Thanks much pyrates999!  I use task view a lot, and I really appreciate being able to use it again with E.P.

**Reply by DrMartyG** [reply]
Just when I thought all was well with Explorer Patcher, after unticking "Get the latest updates as soon as available" to avoid any preview updates being installed, Windows updates 2023-08 Cumulative Update for .NET Framework 3.5 and 4.8.1 for Windows 11, version 22H2 for x64 (KB5028948) and 2023-08 Cumulative Update for Windows 11 Version 22H2 for x64-based Systems (KB5029263) got installed  today, and Explorer Patcher is crashing for me no matter which style task bar I use.

**Reply by Bassadin** [reply]
Same here, I'm now also on Build 22621.2134 and it crashes like it did on my preview version :)

**Reply by pyrates999** [reply]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by Bassadin** [reply]
Nope, same thing as before. Even after following your steps, Task View only seems to work when using the Win 11 Taskbar layout, and when using that layout, right-clicking the taskbar crashes and restarts `explorer.exe`.

**Reply by pyrates999** [reply]
Please wait for it to be fixed then.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1828

Please close this one.

**Reply by jp-d-cruz** [reply]
https://github.com/valinet/ExplorerPatcher/issues/1828 is about file explorer crash. this forum is about task view being disabled. I'm also interested in this form as my task view stopped working recently BUT my file explorer works.

**Reply by pyrates999** [reply]
You're right, sorry about that.

duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1687

Please close this one.

Work around: set to use windows 11 taskbar instead of windows 10 taskbar.

**Comment by jp-d-cruz** [comment]
Task view works when i uninstall EP. But i don't like the taskbar layout of windows 11 so i'm trading the lost of task view functionality for the more customizable taskbar. Will wait for any fix for it. TY for the hard work of people concerned.
PS. task view works when i set "Taskbar Style" in EP properties to Windows 11.. but setting it to the more preferred prior windows taskbar styles will disable task view.

**Comment by Poopooracoocoo** [comment]
Closing as duplicate of https://github.com/valinet/ExplorerPatcher/issues/1687 and https://github.com/valinet/ExplorerPatcher/issues/1802
